streams/src/test/java/org/apache/kafka/streams/state/internals/ChangeLoggingKeyValueBytesStoreTest.java

```diff
@@ -280,12 +276,9 @@ private StreamsConfig streamsConfigMock() {
 
         final Map<String, Object> myValues = new HashMap<>();
         myValues.put(InternalConfig.IQ_CONSISTENCY_OFFSET_VECTOR_ENABLED, true);
-        expect(streamsConfig.originals()).andStubReturn(myValues);
-        expect(streamsConfig.values()).andStubReturn(Collections.emptyMap());
-        expect(streamsConfig.getString(StreamsConfig.APPLICATION_ID_CONFIG)).andStubReturn("add-id");
-        expect(streamsConfig.defaultValueSerde()).andStubReturn(Serdes.ByteArray());
-        expect(streamsConfig.defaultKeySerde()).andStubReturn(Serdes.ByteArray());
```

Comment:
Are these ever invoked? Should we stub and/or verify these calls? 🤔

Reply:
I followed the same reasoning as in the other comment for removed stubs - Mockito reported these as extraneous and deleting them from EasyMock tests caused tests to still pass.

Reply:
I can also confirm I had the same problem when migrating `ProcesserStateManagerTest` in https://github.com/apache/kafka/pull/12524, i.e. there was quite a few tests which had extra/unnecessary stubs/verifications which `MockitoJUnitRunner.StrictStubs` picked up.